### PR TITLE
Public Releases of ANT SDK v.C.B7 and ANT+ SDK v.P.B6

### PR DESCRIPTION
**Pull request by harrychin**
AntLib and PluginLib have been updated to support API Level 30 and higher. This is due to a change in how Android handles package visibility between apps.

All sample apps have been converted to Gradle.

All apps support Android 13 and API Level 33.

### CODE REVIEW

File: ANT+_Android_SDK/README.md [hunk 1]
```diff
@@ -160,35 +201,36 @@ Android ANT+ SDK Changelog
...
-> 
-> 
+> * Now releasing through GitHub in addition to thisisant.com (<https://github.com/ant-wireless/ANT-Android-SDKs>)
+>
+>
 > Ant+ Plugins Service and ANT+ PluginLib Changelog - 18 June 2014 - v.3.0.0 RC1
 > -------------------------------------------------------------
-> 
+>
 > ###New Features and Improvements
```
Comment (Scault) [inline ANT+_Android_SDK/README.md:232]: nit: Won't format properly without a space between "###" and "Features".

File: ANT+_Android_SDK/README.md [hunk 2]
```diff
@@ -205,16 +247,16 @@ Android ANT+ SDK Changelog
 > * Minor improvements to some function signatures and names to match profiles and common usages
 > <br>
 > <br>
-> ###Exclusive New Features for ANT+ Members 
+> ###Exclusive New Features for ANT+ Members
```
Comment (Scault) [inline ANT+_Android_SDK/README.md:250]: nit: Here too.

File: ANT+_Android_SDK/README.md [hunk 3]
```diff
@@ -205,16 +247,16 @@ Android ANT+ SDK Changelog
...
 > ####<i>(ANT+ Members get exclusive access to new profiles for a given period of time, will be released publically at a later time)</i>
```
Comment (Scault) [inline ANT+_Android_SDK/README.md:251]: nit: Here too.

File: ANT+_Android_SDK/README.md [hunk 4]
```diff
@@ -205,16 +247,16 @@ Android ANT+ SDK Changelog
...
-> 
+>
 > * Add Fitness Equipment Controls (including trainer, stationary bike, and open broadcast connections) support
 > * Add support for bike speed and cadence new pages 4 and 5
 > * Add support for battery identifier and number of batteries information to battery status event
 > <br>
 > <br>
 > ###Bug Fixes
```
Comment (Scault) [inline ANT+_Android_SDK/README.md:258]: nit: Here too.

File: ANT_Android_SDK/README.md [hunk 5]
```diff
@@ -108,59 +145,65 @@ Android ANT SDK Changelog
...
-> * Now releasing through GitHub in addition to thisisant.com (https://github.com/ant-wireless/ANT-Android-SDKs)
-> 
-> 
+> * Now releasing through GitHub in addition to thisisant.com (<https://github.com/ant-wireless/ANT-Android-SDKs>)
+>
+>
 > ANT Radio Service and ANTLib - 18 June 2014 - v.4.5.0 (RC7, RC4)
 > -------------------------------------------------------------
+>
 > ###New Features and Improvements
```
Comment (Scault) [inline ANT_Android_SDK/README.md:207]: nit: Won't format properly without a space between "###" and "Features".

Comment (Scault) [inline ANT_Android_SDK/README.md:207]: nit: Won't format properly without a space between "###" and "Features". Same with line 218.

**Review by Scault — COMMENTED** [review]
Just a couple of nits.

Everything that requires updating has been updated - looks good!

**Review by Scault — APPROVED** [review]

**Review by DEVFLIP767 — APPROVED** [review]

**Pull request merged**